### PR TITLE
[Empty] box now appears

### PR DESCRIPTION
**Pull request by tstrother**
When a user was assigned to all projects, selecting "Assign User to Project" would just give no option. Now, it shows an unselectable [Empty] box similar to when you try to remove a user from a project when they are not in any projects.

### CODE REVIEW

**Comment by fallinbryan** [comment]
👍

File: source/views/teamManagerView.py [hunk 1]
```diff
@@ -72,6 +72,8 @@ def generateMemberMenus(self):
 
     def generateAddToProjectMenus(self):
         projectOptions = tk.Menu(self.memberPopupMenu, tearoff=0, cursor = "hand2")
+        if self.inspectedItem.listOfProjects:
```
Comment (Kroronos) [inline source/views/teamManagerView.py]: No bad this doesn't work. 
Exception in Tkinter callback
Traceback (most recent call last):
  File "C:\Users\astro\Documents\GitHub\GroupProject\source\data\ScrumblesData.py", line 95, in setData
    self.dbConnection.commit()
_mysql_exceptions.OperationalError: (2006, 'MySQL server has gone away')

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\astro\AppData\Local\Programs\Python\Python36-32\lib\tkinter\__init__.py", line 1702, in __call__
    return self.func(*args)
  File "C:\Users\astro\Documents\GitHub\GroupProject\source\views\teamManagerView.py", line 90, in <lambda>
    lambda project=project: self.controller.dataBlock.removeUserFromProject(project, self.inspectedItem))
  File "C:\Users\astro\Documents\GitHub\GroupProject\source\data\DataBlock.py", line 9, in wrapper
    func(self,*args)
  File "C:\Users\astro\Documents\GitHub\GroupProject\source\data\DataBlock.py", line 271, in removeUserFromProject
    self.conn.setData(ProjectQuery.removeUser(project,user))
Exception in thread Thread-2:
Traceback (most recent call last):
  File "C:\Users\astro\AppData\Local\Programs\Python\Python36-32\lib\threading.py", line 916, in _bootstrap_inner
    self.run()
  File "C:\Users\astro\AppData\Local\Programs\Python\Python36-32\lib\threading.py", line 864, in run
    self._target(*self._args, **self._kwargs)
  File "C:\Users\astro\Documents\GitHub\GroupProject\source\data\DataBlock.py", line 431, in updater
    self.cv.wait_for(self.updateAllObjects)
  File "C:\Users\astro\AppData\Local\Programs\Python\Python36-32\lib\threading.py", line 321, in wait_for
    result = predicate()
  File "C:\Users\astro\Documents\GitHub\GroupProject\source\data\DataBlock.py", line 117, in updateAllObjects
    projectTable = self.conn.getData(Query.getAllProjects)
  File "C:\Users\astro\Documents\GitHub\GroupProject\source\data\ScrumblesData.py", line 60, in getData
    self.dbConnection.query(query)
  File "C:\Users\astro\AppData\Local\Programs\Python\Python36-32\lib\site-packages\MySQLdb\connections.py", line 277, in query
    _mysql.connection.query(self, query)
_mysql_exceptions.OperationalError: (2006, 'MySQL server has gone away')

  File "C:\Users\astro\Documents\GitHub\GroupProject\source\data\ScrumblesData.py", line 98, in setData
    self.dbConnection.rollback()
_mysql_exceptions.OperationalError: (2006, 'MySQL server has gone away')
Exception in Tkinter callback
Traceback (most recent call last):
  File "C:\Users\astro\AppData\Local\Programs\Python\Python36-32\lib\tkinter\__init__.py", line 1702, in __call__
    return self.func(*args)
  File "C:\Users\astro\Documents\GitHub\GroupProject\source\views\teamManagerView.py", line 39, in <lambda>
    self.memberList.listbox.bind('<2>' if self.aqua else '<3>', lambda event: self.memberPopup(event))
  File "C:\Users\astro\Documents\GitHub\GroupProject\source\views\teamManagerView.py", line 123, in memberPopup
    self.generateMemberMenus()
  File "C:\Users\astro\Documents\GitHub\GroupProject\source\views\teamManagerView.py", line 70, in generateMemberMenus
    self.generateAddToProjectMenus()
  File "C:\Users\astro\Documents\GitHub\GroupProject\source\views\teamManagerView.py", line 75, in generateAddToProjectMenus
    if self.inspectedItem.listOfProjects:
AttributeError: 'str' object has no attribute 'listOfProjects'
Exception in Tkinter callback
Traceback (most recent call last):
  File "C:\Users\astro\AppData\Local\Programs\Python\Python36-32\lib\tkinter\__init__.py", line 1702, in __call__
    return self.func(*args)
  File "C:\Users\astro\Documents\GitHub\GroupProject\source\views\teamManagerView.py", line 39, in <lambda>
    self.memberList.listbox.bind('<2>' if self.aqua else '<3>', lambda event: self.memberPopup(event))
  File "C:\Users\astro\Documents\GitHub\GroupProject\source\views\teamManagerView.py", line 123, in memberPopup
    self.generateMemberMenus()
  File "C:\Users\astro\Documents\GitHub\GroupProject\source\views\teamManagerView.py", line 70, in generateMemberMenus
    self.generateAddToProjectMenus()
  File "C:\Users\astro\Documents\GitHub\GroupProject\source\views\teamManagerView.py", line 75, in generateAddToProjectMenus
    if self.inspectedItem.listOfProjects:
AttributeError: 'str' object has no attribute 'listOfProjects'
Exception in Tkinter callback
Traceback (most recent call last):
  File "C:\Users\astro\AppData\Local\Programs\Python\Python36-32\lib\tkinter\__init__.py", line 1702, in __call__
    return self.func(*args)
  File "C:\Users\astro\Documents\GitHub\GroupProject\source\views\teamManagerView.py", line 39, in <lambda>
    self.memberList.listbox.bind('<2>' if self.aqua else '<3>', lambda event: self.memberPopup(event))
  File "C:\Users\astro\Documents\GitHub\GroupProject\source\views\teamManagerView.py", line 123, in memberPopup
    self.generateMemberMenus()
  File "C:\Users\astro\Documents\GitHub\GroupProject\source\views\teamManagerView.py", line 70, in generateMemberMenus
    self.generateAddToProjectMenus()
  File "C:\Users\astro\Documents\GitHub\GroupProject\source\views\teamManagerView.py", line 75, in generateAddToProjectMenus
    if self.inspectedItem.listOfProjects:
AttributeError: 'str' object has no attribute 'listOfProjects'

Comment (Kroronos) [reply source/views/teamManagerView.py]: Anyways, besides the error above upon removal.

Checking for the existence of a listOfProjects is insufficient to put empty.
The list of projects must be equivalent to the number of projects that exist.

Comment (fallinbryan) [reply source/views/teamManagerView.py]: Okay,  will fix.  Branched Clean to Sprint3_53

Comment (Kroronos) [reply source/views/teamManagerView.py]: You end up getting stuff like
Assign to project --> [Empty]
                                  Scrumbles
                                  Test

Comment (tstrother) [reply source/views/teamManagerView.py]: I'm getting that error even without those lines of code I added.

Comment (tstrother) [reply source/views/teamManagerView.py]: Not sure if I never tried removing items before or some other thing, but it's now present for me with or without the changes.

**Pull request merged**